pylint/test/unittest_checker_stdlib.py

```diff
@@ -46,3 +47,38 @@ def infer_func(node, context=None):
             ''')
             with self.assertNoMessages():
                 self.checker.visit_call(node)
+
```

Comment:
We should also add two more tests, one for when the argument to `copy.copy` is a different object than `os.environ` and another where the argument is an uninferable object, something along the lines of:

```
from missing_library import MissingObject
copy.copy(MissingObject) # this should not trigger an exception, nor make the checker crash
```

Reply:
Added